multi-int-store/src/test/java/com/connexta/multiintstore/MultiIntStoreIntegrationTest.java

```diff
@@ -93,4 +96,13 @@ public void testRetrieve() throws Exception {
         .andDo(print())
         .andExpect(status().isOk());
   }
+
+  @Test
+  public void testSearchEndpointKeyword() throws Exception {
```

Comment:
Should this method also verify `searchService.find`?
```java
final IndexedProductMetadata indexedProductMetadata =
    new IndexedProductMetadata(
        "1",
        "All the color had been leached from Winterfell until only grey and white remained");
searchService.store(indexedProductMetadata);

assertThat(searchService.find("Winterfell"), contains(indexedProductMetadata));
```

Reply:
Yes, I think it should. That is what the method name purports to test.